rebar.config.erl

```diff
@@ -255,13 +255,13 @@ relx_apps(ReleaseType) ->
     , inets
     , compiler
     , runtime_tools
-    , {emqx, load} % started by emqx_machine
-    , {emqx_conf, load}
     , emqx_machine
-    , {mnesia, load}
-    , {ekka, load}
-    , {emqx_plugin_libs, load}
-    , {esasl, load}
+    , emqx_conf
+    , emqx
+    , mnesia
```

Comment:
we need to check ekka code to see if it allows mnesia to start before it.

Reply:
fixed

Reply:
both `mnesia` and `ekka` will start by `emqx_machine`, so I don't think it would be an issue here